test/test_advent.py

```diff
@@ -0,0 +1,20 @@
+import requests
+
+# TODO: Move the LEADERBOARD_CODE and SESSION_ID someplace else to miniize duplication
```

Comment:
You should, hypothetically be able to import LEADERBOARD_CODE from the actual script.